dbt_subprojects/daily_spellbook/models/_sector/dex/pools/optimism/uniswap/uniswap_pools_optimism_balances.sql

```diff
@@ -0,0 +1,51 @@
+{{
+  config(
+    schema = 'uniswap_pools_optimism',
+    alias = 'balances',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['pool_address', 'snapshot_day'],
+    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.snapshot_day')]
+  )
+}}
+
+with op_addresses as (
+  select      
+    pool as address,  
+    token0,          
+    token1,
+     -- Add a column for token_address where it matches the Optimism token address
+    case
+      when token0 = 0x4200000000000000000000000000000000000042 then token0
+      when token1 = 0x4200000000000000000000000000000000000042 then token1
+    end as token_address,          
```

Comment:
```suggestion
    0x4200000000000000000000000000000000000042 as token_address,
```

Reply:
in order to match your previous setup, we should be able to simply move the hardcoded value here into this CTE? wouldn't this work?